studies/forms.py

```diff
@@ -285,7 +304,6 @@ def __init__(self, *args, **kwargs):
         - Remove empty label from deception/negativity/stressful/risk field and reset the choices
         - mark_safe the labels of negativity/stressful/risk
         """
-        self.study = kwargs.pop("study", None)
```

Comment:
This View is still getting a `study` kwarg argument from somewhere, causing it to error out.